elements/pf-avatar/BaseAvatar.ts

```diff
@@ -26,15 +26,15 @@ export class BaseAvatar extends LitElement {
   @property({ reflect: true }) alt?: string = 'Avatar image';
 
   /** Size of the Avatar */
-  @property({ reflect: true }) size: 'sm'|'md'|'lg'|'xl' = 'sm';
+  abstract size?: string;
```

Comment:
this is, unfortunately, a breaking change, and will thus require a changeset with a `major` version. That also means we're unlikely to merge this quickly